personas/README.md

```diff
@@ -88,25 +88,22 @@
  ### Background:
 - I maintain a couple of small packages and contribute new medium size but impactful features to my underlying ecosystem. (Think a compiler optimisation for floats that takes a few months of work and extremely niche knowledge to get right) This is a really common and critical profile.
 - Diana is in a loose network of other niche people doing the same in my ecosystem.
-- Diana has challenges keeping their toolchain and CI systems up-to-date and running and has antiquated security tools to help (e.g. an out of support MFA token) with no adidtional funds to replace.
+- Diana has challenges keeping their toolchain and CI systems up-to-date and running. C was not made for this kind of work, nor are most of the packaging ecosystem, and they have to fight with them all the time.
```

Comment:
> C was not made for this kind of work

Is `C` the programming language or is it supposed to be `CI`, or something else?

Reply:
Personally, I'd prefer to keep this more generic, since Diana exists in literally _every_ programming language/framework ecosystem, not just C, and dependency hell is a very common time-suck.

Reply:
I put C because it is, by far, the most impacted and the most critical and everyone has to deal with it at some point. But the follow up of the sentence works too. So yes the programming language and yes we can be really open.

Reply:
Fair enough! :)